pkg/controller/client/client_controller.go

```diff
@@ -59,7 +59,9 @@ const (
 	// the Common Services CA certificate and private key details
 	CSCACertificateSecretName string = "cs-ca-certificate-secret"
 	// FinalizerName is the name of the finalizer added to Resources by the Client controller
-	FinalizerName string = "client.oidc.security.ibm.com"
+	FinalizerName     string = "client.oidc.security.ibm.com"
```

Comment:
I prefer `CP3FinalizerName` here since we need to distinguish between the two CS versions.